master/internal/provisioner/scale_decider.go

```diff
@@ -142,31 +141,31 @@ func (s *scaleDecider) findInstancesToTerminate(
 	}
 
 	// Terminate instances that have not connected to the master for a long time.
-	var longUnconnected map[string]bool
-	s.pastDisconnectedInstances, longUnconnected = findInstancesLongInSameState(
+	var longDisconnected map[string]bool
```

Comment:
I don't like calling it 'disconnected'. That implies that it was connected, but the connection has failed, while we are talking about instances that have never connected at all.

Reply:
here is for both unconnected and disconnected instances. So disconnected might be more accurate to describe both cases.

Reply:
Okay, I don't have a better name in mind